pgoapi/pgoapi.py

```diff
@@ -171,7 +171,7 @@ def heartbeat(self):
             self.log.info("Username: %s, Currencies: %s", player_data.get('username', 'NA'), currency_data)
 
         if 'GET_INVENTORY' in res['responses']:
-            with open("data_dumps/%s.json" % self.config['username'], "w") as f:
+            with open("data_dumps/%s.json" % player_data.get('username', 'NA') as f:
```

Comment:
i thought that the login id could be an email which posed a problem when using the web app,  however now this issue wont occur